src/Cake.Common/Text/TextTransformationExtensions.cs

```diff
@@ -29,5 +31,22 @@ public static class TextTransformationExtensions
             transformation?.Template.Register(key, value);
             return transformation;
         }
+
+        /// <summary>
+        /// Registers all keys and values in the dictionary for text transformation.
+        /// </summary>
+        /// <typeparam name="TTemplate">The text transformation template.</typeparam>
+        /// <param name="transformation">The text transformation.</param>
+        /// <param name="tokens">The tokens.</param>
+        /// <returns>
+        /// The same <see cref="TextTransformation{TTemplate}" /> instance so that multiple calls can be chained.
+        /// </returns>
+        public static TextTransformation<TTemplate> WithTokens<TTemplate>(
```

Comment:
I would prefer if we didn't add more methods to the interface since that is an breaking change for the ABI. It would probably be better if we added this as an extension method since it just delegate to the other Register method.

Reply:
Commented on the wrong line.